package.json

```diff
@@ -11,7 +11,8 @@
     "cvat-core",
     "cvat-canvas",
     "cvat-canvas3d",
-    "cvat-ui"
+    "cvat-ui",
+    "tests"
```

Comment:
Removed from ```package.json```. Added to ```eslint.yml```. 
Otherwise I get an eslint error like this for example
```"Unable to resolve path to module 'cy-verify-downloads'."```